### ISSUE TITLE
Add curl option for downloading available references

### ISSUE THREAD

**Issue by ayoraind** [rki-mf1/clean#104]
Hi team,

Thanks for your awesome pipeline. I have a suggestion: could you kindly add the possibility to download reference files via curl too? I got an `Unable to establish SSL connection` error message when running the pipeline (caused by wget). I see that the curl option can be added easily (https://github.com/rki-mf1/clean/blob/main/modules/prepare_contamination.nf#L19-L48). Initially, I forked the repository and edited the prepare_contamination.nf file (https://github.com/ayoraind/clean/blob/main/modules/prepare_contamination.nf#L19-L48). The pipeline worked perfectly after the change. However, I think it is best to include both wget and curl so that if wget fails due to some weird errors, curl would not.

**Comment by MarieLataretu** [comment]
Hi @ayoraind , thanks for the suggestion!

Could you try the following to see if it works for you?

```
nextflow pull rki-mf1/clean
nextflow run rki-mf1/clean -r feature_run-curl-if-wget-fails [...]
```

**Comment by ayoraind** [comment]
Hi @MarieLataretu

Thanks for your response. I was unable to `nextflow pull` the pipeline repository due to firewall issues. 
`WARN: Cannot read project manifest -- Cause: Unable to tunnel through proxy. Proxy returns "HTTP/1.1 407 Proxy Authentication Required"`

I used `git clone` instead. I got the error `Revision option cannot be used when running a local script` when I used `-r feature_run-curl-if-wget-fails`
